apps/site/src/components/icons/sanity-icon.tsx

```diff
@@ -0,0 +1,29 @@
8iFnfOrVgDBFRwcif9cdZH5YqoLCCTGrBHp5UhUfWnArY3/dIHpBWGiCLbL3Nw5v2omEazh2U+k75CNONFlTnSgufFtpGb8QAzAT0INay5xoKkFbZK86y50f2dt9QSfoZZaWp/OLxrn1ZYZtwsd6RCTdywumgYgAfRa0VmlDXesdoh5YkYCARFsFw8HvsHGUHizoXAAX4ggsW0XOzOPpvPJRUPZUvD3XtLfLlu6ZmMezZGm90BABNvlw6PlmuoDA1FYDQjTCBAkv7bLHZpL85F3giCwVkeJfrzF0AsOumVtay5NkLaSIyCCnRwvxz7dVlt9qYmwTBOV8k9cCZE41lV5N8zesscHsTh5E8F7WtEVZfV3PJB3w8SAvRIQwd4rIvc80HHVpcPRNFcSwJcNPuyQkPlPHO0eP2bL0sSFxb4v9J8Xhbovx7qf9GSrTak3swTkPc4sT0fU1llbXakQr1MGTiZNtnDHz0mKwx3hopwaEf+1xbNqv2HwIvUrlqbrSupXy6w6p55IvzF5f9Nn6NgaOhdWfxEJf+Az1FFRy+J0mLIw6VhvZd6wuFDzoiJvA9VE2zXpumdCf119et2T0cy3KDVmm4AIdrYJ57n+5y+/3HdwmbrYh3g1ERxkx7f7tgHm2TRpPssE7Dg1ALBgR4nu7dW4UO5czDL0LFcvgp1lwE6qvq22ZqECmgtAI9guWZx0kncyY0viScW+GpssUDeUN6x+LjMtSC35JCCCnU/6eWib6mpCHZ10NWr6FgCM4f23vIc78cIEGRR5cEyKTe7sN55N8y8oTXA/WLSs9LY7nkqxainmQALybjrQKbkwadvVl5cHtTEVAGaZhnEqaoKwvavEftntjwyOXHgitTZsFxFvz+NrFXkx0Y5R84LigwrV7cElqx5JrWYp5WQC8k462Ts5sq3zmtnHUjQ6y6fwKwqxONJ3YUJ8W6AMkhw5oh/N7FhIVHxVF9rxaQBo741a96LPtyp0y4o/96MaecSlBORddKnjsmH21jmXjwwYxiUmwkxE3Jd/WvPszc5Vwocs5JN3Aqbi28rjM2v1Udiyftaj1PJhS1d/lHfjxICsExDBzjpidzbQPm/mRQrVNwDhOCIaCX0n4+ILlRIyyb5f41+R/JLaV8TFvkA7CKlZE/6qrL7xf7JvhbTgJAIi2E7yhgNt+XT27DJ/QJ+DoKcD0HQEGKpQ2TlLdnVpmQyo1Jy48+8X5sjxaYNZx37hvISA/xtR5q8HLF3xVmqtSCm3E5D3y+0ezKH97dfOHuqzohVEOJ2IvhowjVIWE45582fHHu/4ihhylFU+uyIQR2T/UuGFw75ZNP87YJoQtiywNL0RMNWvolr9MfjOR5vwvvssoVnYBOR9Kmz/p9z7LTU1odIQHB/V+iID1bSgzxylLe23xbtPgFjMeedC4k/7lBv0QMHEWTQvFvLBFhZrrQl8hgGGoSI94cgbgPgLC2FdQ/Gwv9XV1cntyx7wfaa6IIKdKZIFXE9dXZ2a3bZlHxP1KQrgXJ+hJiDgKEvrEhYpFidWHVu8C3Txkn9rcIiDRTp2L6fmcIeFCt8PR6w/E9EDnYSPjbx1zTsFPJSk63shIIItQyQrBD6dVzPGh3SGAjoFAY8ghHFENJBv4ranlX0hFF5I88JC5q4OHsXuBIgtFvaFQLrQgLeA1D8ssJ7TSA8NWLr2xaw4QCr1JAERbE+61Xmd6p5Ttb9WNF4j7E+AYxXCIUhwCAEcxNZyOth4jhOWt1iqZjvxSV9n+uIsn/3XTn+/c5/7hnYyIzwxsGxXF2uTq/jcomDcpr4YfWL8mWfQFmnQHBlCeIcUvQwWvgxk/YWCgVfKFq38q/O8Ixa5hUAyw9ktfRI7XUKAj8n39tAwHVXDTQVjo0RHAdFxGuhIIBha5Pft2M4WO24dC6nwPuRU7rHc8QWQ1GJo7KgKC7FhhzSUHdoAVLaS90aivGPmA0R8FQFe8Zvmn8NkvavC8FGg3NiCdSvaXOIOMdMFBESwXeCkQjORZs8ObAMI+EMQoK7oIPTRPoRqX9Q0GhTsgxr2IdBjgqZvmGEaASDy8S44nqjzv4n4f5MBsX8nhF9imQrtPc2fn7DHECNqRIwAQAQQwwAUDocjvRGttyOqjwj1ewj4NoF627DoXTDNf/WGjY4BAwZ0YV1duND8JP3NPQER7NwzlxYzSOD5CRN84QMPNIcPBH9Z8cBiC6JFPssX1GgVGcoMKrCKSJnFCBBQaBWDRkUKo4TQS1EMKxMiRDpMWoejoDqtiPVpr9XVsvXZv3VMfOEFzhktRzwz6C+pKj0CItjp8ZPSQkAICIGcERDBzhlqaUgICAEhkB4BEez0+ElpISAEhEDOCIhg5wy1NCQEhIAQSI+ACHZ6/KS0EBACQiBnBESwc4ZaGhICQkAIpEdABDs9flJaCAgBIZAzAiLYOUMtDQkBISAE0iMggp0ePyktBISAEMgZARHsnKGWhoSAEBAC6REQwU6Pn5QWAkJACOSMgAh2zlBLQ0JACAiB9AiIYKfHT0oLASEgBHJGQAQ7Z6ilISEgBIRAegREsNPjJ6WFgBAQAjkjIIKdM9TSkBAQAkIgPQIi2Onxk9JCQAgIgZwREMHOGWppSAgIASGQHgER7PT4SWkhIASEQM4IiGDnDLU0JASEgBBIj4AIdnr8pLQQEAJCIGcERLBzhloaEgJCQAikR0AEOz1+UloICAEhkDMCItg5Qy0NCQEhIATSIyCCnR4/KS0EhIAQyBkBEeycoZaGhIAQEALpERDBTo+flBYCQkAI5IyACHbOUEtDQkAICIH0CIhgp8dPSgsBISAEckZABDtnqKUhISAEhEB6BESw0+MnpYWAEBACOSMggp0z1NKQEBACQiA9AiLY6fGT0kJACAiBnBEQwc4ZamlICAgBIZAeARHs9PhJaSEgBIRAzgiIYOcMtTQkBISAEEiPgAh2evyktBAQAkIgZwT+P9IwWPMnYkCOAAAAAElFTkSuQmCC"
```

Comment:
Seems like we've got a raster image inside an SVG here. Is there a way to find a truly vector logo? Not a big fan of JSX SVGs because they increase bundle size, but if we do this, it's best to avoid base64 in them if possible.

Reply:
I agree, this is what figma exported for me. I'll try to get the right svg

Reply:
Done in 78774dcaf2aeccf53210e67ecabadfb2a6b70639